Src/Sarif.PatternMatcher/ValidatorsCache.cs

```diff
@@ -189,8 +189,10 @@ public static void DisableValidationCaching(MethodInfo shouldCacheMethodInfo, bo
             var results = new List<ValidationResult>();
             combinations ??= GetCombinations(mergedGroups);
 
+            string filePath = context.TargetUri.GetFilePath();
             var flexMatchProperties = new Dictionary<string, FlexMatch>();
             flexMatchProperties.AddProperties(properties);
+            flexMatchProperties["scanTargetFullPath"] = new FlexMatch { Value = filePath };
```

Comment:
>flexMatchProperties [](http://example.com/codeflow?start=12&length=19)

we can still do better than this, but you improved things!
i will go tinker w/the code next, will do this early next week.